docs/src/man/comparisons.md

```diff
@@ -1,77 +1,163 @@
 # Comparisons
 
-This section compares DataFrames.jl with other data manipulation frameworks.
+This section compares DataFrames.jl with other data manipulation frameworks in Python, R, and Stata.
+
+A sample data set can be created using the following code:
+
+```julia
+df = DataFrame(id = 'a':'f', grp = [1,2,1,2,1,2], x = 6:-1:1, y = 4:9, z = 3:8)
+```
+
+## Comparison with the Python package pandas
+
+The following table compares the main functions of DataFrames.jl with the Python package pandas (version 1.1.0):
+
+```python
+df = pd.DataFrame(
+        {'grp': [1,2,1,2,1,2], 'x': range(6,0,-1), 'y': range(4,10), 'z': range(3,9)},
+        index = list('abcdef'))
+```
+
+Note that pandas allows row indexing by labels. For DataFrames.jl, you can achieve the same result via row filtering as shown below.
+
+### Accessing data
+
+| Operation                  | Example                     | pandas                 | DataFrames.jl            |
+|:---------------------------|:----------------------------|:-----------------------|:-------------------------|
+| Cell indexing by location  | Cell at row 2, column 2     | `df.iloc[1, 1]`        | `df[2, 2]`               |
```

Comment:
Mention 1-based indexing above (or below) this table? Not saying anything about it could be confusing.

Reply:
I will add a note below the table.

Reply:
Done